SPECS-SIGNED/fwctl-hwe-signed/fwctl-hwe-signed.spec

```diff
@@ -120,6 +120,9 @@ fi # 1 : closed
 %config(noreplace) %{_sysconfdir}/depmod.d/zz02-fwctl-*.conf
 
 %changelog
+* Wed Nov 05 2025 Siddharth Chintamaneni <sidchintamaneni@gmail.com> - 24.10-24_6.12.57.1-1
```

Comment:
nit: for the future we should make the changelog reflect the rpm's versioning. 

24.10-24_6.12.57.1**-1**
24.10-24_6.12.57.1**.1**